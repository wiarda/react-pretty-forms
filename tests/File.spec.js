/* global describe test expect */
import React from 'react';
import renderer from 'react-test-renderer';
import { FormFile } from '../index';

describe('File component tests', () => {
  test('Renders consistently', () => {
    const component = renderer
      .create(<FormFile name="Upload" required label="contacts" />)
      .toJSON();
    expect.assertions(1);
    expect(component).toMatchSnapshot();
  });
});
