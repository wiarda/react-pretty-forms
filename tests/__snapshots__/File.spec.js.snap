// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`File component tests Renders consistently 1`] = `
<div
  className="pretty-row"
>
  <label
    className="pretty-label"
    data-active={false}
    data-validity={true}
    htmlFor="Upload"
  >
    <span
      className="pretty-label-text"
      data-active={false}
    >
      contacts
    </span>
    <input
      accept=".csv,.xls"
      className="pretty-file"
      data-validity={true}
      multiple={false}
      name="Upload"
      required={true}
      type="file"
    />
  </label>
  <div
    className="pretty-validation"
    data-validity={true}
  >
    Please fill in this required field
  </div>
</div>
`;
